src/server/cli/build-settings.ts

```diff
@@ -48,9 +50,10 @@ export default function buildSettings(config: object, options: ServerOptions, au
     ...options
   };
   const serverSettings = ServerSettings.fromJS(serverSettingsJS);
-  const appSettings = appSettingsFromConfig(config);
+  const logger = getLogger(serverSettings.loggerFormat);
+  const appSettings = appSettingsFromConfig(config, logger);
```

Comment:
All `fromConfig` methods require now logger as explicit dependency. Maybe we could use some `WriterMonad` like pattern but that's a story for another time/PR.